test/fixtures/resolutions/yarn1/expected-tree.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "adm-zip": {
+      "labels": {
+        "scope": "prod"
+      },
+      "name": "adm-zip",
+      "version": "0.4.8"
```

Comment:
According to [this](https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md#relation-to-non-nested-dependencies), direct dependencies (dev, regular, optional) will always take precedence over the `resolutions`